lib/pages/dashboard_page.dart

```diff
@@ -61,9 +60,16 @@ class _DashboardPageState extends State<DashboardPage> {
                   ),
                   InkWell(
                     onTap: () {
-                      setState(() => selectedItem = 0);
+                      setState(() {
+                        selectedItem = 0;
+                        selectedItemList[0] = true;
+                        selectedItemList[1] = false;
+                        selectedItemList[2] = false;
+                        selectedItemList[3] = false;
```

Comment:
Not sure what it is, why do you change this state?

Reply:
This is for designing purpose. So that when one menuItem is selected, it's color changes so we know it's the one selected. And when one menuItem is selected, others should be unselected.